provision/acc_provision/acc_provision.py

```diff
@@ -911,8 +912,12 @@ def provision(args, apic_file, no_random):
     }
     if args.username:
         config["aci_config"]["apic_login"]["username"] = args.username
-    if args.password:
-        config["aci_config"]["apic_login"]["password"] = args.password
+
+    try:
+        config["aci_config"]["apic_login"]["password"] = os.environ['ACC_PROVISION_PASS']
+    except KeyError:
+        if args.password:
+            config["aci_config"]["apic_login"]["password"] = args.password
```

Comment:
This whole chunk of code can simplified to:

    config["aci_config"]["apic_login"]["password"] = \
            os.environ.get('ACC_PROVISION_PASS', args.password)

Reply:
Should the command line parameter not override the one  in environment (that was a problem with the original patch as well)? I would recommend that it be something like:
'''
config["aci_config"]["apic_login"]["password"] = \
    args.password if args.password else os.environ.get('ACC_PROVISION_PASS')
'''

Reply:
Much nicer thanks! Update following shortly.